docs/maui/includes/validation-behavior.md

```diff
@@ -0,0 +1,15 @@
+### ValidationBehavior Properties
+
+The following properties are implemented in `public abstract class ValidationBehavior`:
```

Comment:
Looking at it again, we should probably note that the current `Behavior` implements `ValidationBehavior`. Something like this?

```suggestion
The following properties are implemented in the base class, `public abstract class ValidationBehavior`:
```

Reply:
@bijington thoughts about this last comment here?